### ISSUE TITLE
Locally hosted images?

### ISSUE THREAD

**Issue by robputer** [huiseoul/react-native-fit-image#35]
<Image> supports 
`source={require('./img/favicon.png')}`

Wonder if it's possible to add support for local images instead of always grabbing from the web?

**Comment by steffenmllr** [comment]
@robputer if you don't need remote images you can achieve the same effect like this:

``` jsx
import React from 'react';
import { Image, Dimensions } from 'react-native';

export default class FitImage extends React.Component {
    constructor(props) {
        super(props);
        this.state = this._getSize(props);
    }
    _getSize(props) {
        const deviceWidth = Dimensions.get('window').width;
        const deviceHeight = Dimensions.get('window').height;
        const ratio = Math.min(deviceWidth / props.originalWidth, deviceHeight / props.originalHeight);

        const width = deviceWidth;
        const height = ratio * props.originalHeight;
        return {width, height};
    }

    resize() {
        this.setState(this._getSize(this.props));
    }

    render() {
        const {width, height} = this.state;
        return (
            <Image
                style={{flex: 1, width, height}}
                source={this.props.source}
                onLayout={(event) => this.resize(event)}
            />
        );
    }
}
```

```
<FitImage
    source={Images.localImage}
    originalWidth={1440}
    originalHeight={830}
/>
```

**Comment by originerd** [comment]
Fixed on v1.4.1
